DataProcessing/datax-host/pom.xml

```diff
@@ -51,7 +51,7 @@ SOFTWARE
 
     <groupId>com.microsoft.datax</groupId>
     <artifactId>datax-host_2.4_2.11</artifactId>
-    <version>1.2.12-SNAPSHOT</version>
+    <version>1.2.13-SNAPSHOT</version>
```

Comment:
missing version rev for other dependency datax package. like datax-core

Reply:
just checked. looks like the {version} is being used for the dependencies so its good